RadarSDK/RadarLocationManager.m

```diff
@@ -26,13 +26,16 @@ @interface RadarLocationManager ()
 @property (assign, nonatomic) BOOL scheduled;
 @property (strong, nonatomic) NSTimer *timer;
 @property (nonnull, strong, nonatomic) NSMutableArray<RadarLocationCompletionHandler> *completionHandlers;
+@property (nonnull, strong, nonatomic) NSMutableSet<NSString *> *nearbyBeaconIdentifers;
 
 @end
 
 @implementation RadarLocationManager
 
-static NSString *const kRegionIdentifer = @"radar";
-static NSString *const kRegionSyncIdentifer = @"radar_sync";
+static NSString *const kIdentifierPrefix = @"radar_";
+static NSString *const kBubbleGeofenceIdentifierPrefix = @"radar_bubble_";
```

Comment:
any one-time upgrade issues that might lead to orphaning regions with old identifiers that are being monitored?

Reply:
All legacy regions are prefixed with `radar_`, which is the prefix we use when removing on `stopTracking()`, so we're good